src/main/java/Duke.java

```diff
@@ -1,25 +1,54 @@
 import java.util.Scanner;
 
 public class Duke {
-    public static void main(String[] args) {
 
-        boolean isByeEntered = true;
-        String outputs;
+    public static final int MAX_INT = 100;
 
+    public static void greeting() {
         System.out.println(" Hello! I'm Duke");
         System.out.println(" What can I do for you?");
 
-        while(isByeEntered) {
+    }
+
+
+
+    public static void add() {
+        boolean isByeEntered = true;
+        String outputs;
+        String[] storeList;
+        storeList = new String[MAX_INT];
+        int i = 0;
+        int a = 1;
```

Comment:
I wasn't sure on what `a` did until I read the code fully... I think changing the variable name `a` could make it more understandable for readers!